.github/workflows/nightly_looped_sanitizers.yml

```diff
@@ -0,0 +1,80 @@
+# Check code with looped compilers' sanitizers
+# This build lasts 6 hours.
+name: Looped_Sanitizers
+
+# This job is run at 00:00 UTC every day or on demand.
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
this would block a lot of runners every night, perhaps only once a week - `'0 1 * * 6'` - Saturday at 1am

Reply:
Done